examples/datasource-logs/src/types.ts

```diff
@@ -0,0 +1,27 @@
+import { DataSourceJsonData } from '@grafana/data';
+import { DataQuery } from '@grafana/schema'
+
+export interface MyQuery extends DataQuery {
+  queryText: string;
+  limit: number;
+}
+
+export const DEFAULT_QUERY: Partial<MyQuery> = {
+  queryText: '',
+  // default limit is 100
+  limit: 100,
+};
+
+/**
+ * These are options configured for each DataSource instance
+ */
+export interface MyDataSourceOptions extends DataSourceJsonData {
+  path?: string;
+}
+
+/**
+ * Value that is used in the backend, but never sent over HTTP to the frontend
+ */
+export interface MySecureJsonData {
+  apiKey?: string;
```

Comment:
I believe this can be removed because secure json is not being used (the datasource is frontend-only)

Reply:
Fixed in https://github.com/grafana/grafana-plugin-examples/commit/50cb0402d43df9724ee84135a2a19839ade2da35